Data-Structures/Linked-List/SinglyLinkedList.js

```diff
@@ -274,6 +274,20 @@ class LinkedList {
   log () {
     console.log(JSON.stringify(this.headNode, null, 2))
   }
+
+  // Method to reverse the LinkedList where each node contains a single integer value
+  reverseList () {
```

Comment:
You may omit the redundant `List` from the method name.

Reply:
Done.